Keas.Mvc/ClientApp/components/Equipment/AssignEquipment.tsx

```diff
@@ -102,39 +102,52 @@ export default class AssignEquipment extends React.Component<IProps, IState> {
                     onSelect={this._onSelectPerson}
                   />
                 </div>
-
+                {!this.state.equipment &&
                 <div className="form-group">
-                  <label>Pick an equipment to assign</label>
                   <SearchEquipment
                     selectedEquipment={this.state.equipment}
                     onSelect={this._onSelected}
                     onDeselect={this._onDeselected}
                     space={this.props.space}
                     openDetailsModal={this.props.openDetailsModal}
                   />
-                </div>
-                {!this.state.equipment ||
-                  (!this.state.equipment.teamId && ( // if we are creating a new equipment, edit properties
-                    <EquipmentEditValues
-                      selectedEquipment={this.state.equipment}
-                      commonAttributeKeys={this.props.commonAttributeKeys}
-                      changeProperty={this._changeProperty}
-                      disableEditing={false}
-                      updateAttributes={this._updateAttributes}
-                      creating={true}
-                      space={this.props.space}
-                      tags={this.props.tags}
-                    />
-                  ))}
+                </div>}
                 {this.state.equipment &&
-                  !!this.state.equipment.teamId && (
-                    <EquipmentEditValues
-                      selectedEquipment={this.state.equipment}
-                      commonAttributeKeys={this.props.commonAttributeKeys}
-                      disableEditing={true}
-                      creating={true}
-                      tags={this.props.tags}
+                  !this.state.equipment.teamId && ( // if we are creating a new equipment, edit properties
+                    <div>
+                    <div className="row justify-content-between">
+                      <h3>Create New Equipment</h3>
+                      <Button className="btn btn-link" onClick={() => this._onDeselected()}>
```

Comment:
FYI if you don't have any params to pass you can always shorthand to just `onClick={this._onDeselected}` no need for the anon func